fake_track_generator: 
  lavacons: [ # world frame
     0, -1,
     0, 1,
     5, -1,
     5, 1,
     10, -1,
     10, 1,
     12, 2,
     12, 0,
     15, 2,
     15, 5,
     17, 5,
     15, 7.5,
     17, 7.5,
     15, 10,
     17, 10,
     13.5, 3.5,
  ]
  max_detectable_range: 10 # meter
